### PR TITLE
Include information about tool maintenance

### PR DESCRIPTION
**Pull request by ChrisJBurns**
- Include information about how to maintain the tool when there is a topic increase

### CODE REVIEW

File: schemas/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Maintenance
+
+## Introduction
+At some point in time there will be an increase an topics or even a change to the existing topic structure. If at any point one of these happen, this will require a change to the Avro schemas in further requiring a change in the structs that represent the Avro schemas. These structs are used by the kafka-management-tool in order to map the unmarshalled JSON message from with the specified topic.
```
Comment (jsimons1989) [inline schemas/README.md]: These doesn't look correct:

- "an increase an topics"
- "JSON message from with the specified topic"

File: schemas/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Walkthrough
+If a change is made to an existing topic structure/schema or if a new topic and Avro schema has been added this will need to be reflected in the kafka-management-tool schema struct for the corresponding Avro schema.
+
+### Addition of new topic and Avro schema
+If a new topic is to be added, there will need to be a new Avro schema file that represents the message structure. Additionally, the services that produce and consume messages from the new topic will need to create the mapping object/struct that represents the topic schema. In `Java` services this is a `Pojo` and in `Go`, a `struct`. For information on how the struct will can be created for the kafka-management-tool look below.
+Additionally, once the new struct has been created, another condition will need to be added to the `schemas.go` file to make sure that the correct struct is used when the new topic is specified in the tool.
```
Comment (jsimons1989) [inline schemas/README.md:11]: I would find this easier to read if the example was with the explanation.

File: schemas/README.md (hunk 1 above)
Comment (ChrisJBurns) [reply schemas/README.md]: ahh yes, meant to be `in`... good spot

File: schemas/README.md (hunk 2 above)
Comment (ChrisJBurns) [reply schemas/README.md:11]: Done

File: schemas/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Maintenance
+
+## Introduction
+At some point in time there will be in increase an topics or even a change to the existing topic structure. If at any point one of these happen, this will require a change to the Avro schemas in further requiring a change in the structs that represent the Avro schemas. These structs are used by the kafka-management-tool in order to map the unmarshalled JSON message from with the specified topic.
+
+## Walkthrough
+If a change is made to an existing topic structure/schema or if a new topic and Avro schema has been added this will need to be reflected in the kafka-management-tool schema struct for the corresponding Avro schema.
+
+### Addition of new topic and Avro schema
+If a new topic is to be added, there will need to be a new Avro schema file that represents the message structure. Additionally, the services that produce and consume messages from the new topic will need to create the mapping object/struct that represents the topic schema. In `Java` services this is a `Pojo` and in `Go`, a `struct`. For information on how the struct will can be created for the kafka-management-tool look below.
```
Comment (jsimons1989) [inline schemas/README.md:10]: Needs a slight rewording: "how the struct will can be created"

File: schemas/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      \"type\": \"string\"
+    }
+  ]
+}
+```
+
+The `Go` struct will look like the following
+
+```go
+type TxClosed struct {
```
Comment (jsimons1989) [inline schemas/README.md:36]: The example above mentions "tx-open" topic but you have called the struct "TxClosed"

File: schemas/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+	Timedate                string  `avro:"timedate"                            json:"timedate"`
+	TransactionDescription  string  `avro:"transaction_details"                 json:"transaction_details"`
```
Comment (jsimons1989) [inline schemas/README.md:38]: I think you should be consistent here and call the variable TransactionDetails

File: schemas/README.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    }
+    {
+      \"name\": \"description\",
+      \"type\": \"string\"
+    }
+  ]
+}
+```
+
+Additionally in the Go structs and/or Java pojos, there would be an extra field added to it to reflect the new field in the Avro schema.
```
Comment (jsimons1989) [inline schemas/README.md:74]: I think this README would be simpler if it only referenced the changes in this repository. It's a bit confusing after this example to mention "in the Go structs and/or Java pojos" when this repo has no Java POJOs and not have an example of the changes to the TxClosed struct.

**Pull request closed without merging**